lib/mix/tasks/compile.adbc_driver.ex

```diff
@@ -0,0 +1,12 @@
+defmodule Mix.Tasks.Compile.AdbcDriver do
+  @moduledoc false
+
+  @doc false
+  def run(_args) do
+    :ok
+  end
+
+  for driver <- Application.compile_env(:adbc, :drivers, [:sqlite, :postgresql]) do
```

Comment:
I don't think we need it as a mix task. I think we can put it at the end of Adbc module itself (or the nif module). And perhaps we should default to an empty list?

Reply:
Okay! I can move it to the end of Adbc module!